app/controllers/api/v1/apps_controller.rb

```diff
@@ -2,7 +2,9 @@ module Api
   module V1
     class AppsController < ApplicationController
       def index
-        apps = App.where('title ilike :prefix', prefix: "#{params[:title]}%")
+        return render json: [] if title.empty?
```

Comment:
был еще вариант написать
```suggestion
        render json: [] and return if title.empty?
```
но многим такое не нравится. я люблю так писать. вот